test/resources/examples/index.js

```diff
@@ -91,149 +91,162 @@ const basicRingTrade = generateTestCase({
   ],
 })
 
-const longRingTrade = generateTestCase({
+const n = 30
+const largeRing30 = generateTestCase({
   name: "Longest Ring Trade",
-  orders: [
-    { sellToken: 0, buyToken: 1, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 1, buyToken: 2, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 2, buyToken: 3, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 3, buyToken: 4, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 4, buyToken: 5, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 5, buyToken: 6, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 6, buyToken: 7, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 7, buyToken: 8, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 8, buyToken: 9, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 9, buyToken: 10, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 10, buyToken: 11, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 11, buyToken: 12, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 12, buyToken: 13, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 13, buyToken: 14, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 14, buyToken: 15, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 15, buyToken: 16, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 16, buyToken: 17, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 17, buyToken: 18, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 18, buyToken: 19, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 19, buyToken: 20, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 20, buyToken: 21, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 21, buyToken: 22, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 22, buyToken: 23, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-    { sellToken: 23, buyToken: 24, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 1 },
-    { sellToken: 24, buyToken: 0, sellAmount: toETH(1), buyAmount: toETH(0.99), user: 0 },
-  ],
+  orders: [...Array(n).keys()]
+    .map(i => ({
+      sellToken: i % n,
+      buyToken: (i + 1) % n,
+      sellAmount: toETH(1),
+      buyAmount: toETH(0.99),
+      user: i % 2,
```

Comment:
nit: would it be easier to understand this test if we just used i as the user and let it run from 0 to 60?

Reply:
There aren't enough accounts loaded into ganache for that.